CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 * Throws a better exception message when calling `RealmObjectSchema.addField()` with a `RealmModel` class (#3388).
 * Use https for Realm version checker (#4043).
-
+* Fixing the scope of the dependencies generated by maven-publish (#5640).
```

Comment:
Perhaps make it more clear what happened, e.g. "Prevent Realms Gradle plugin from transitively forcing specific versions of Google Build Tools onto downstream projects"?